content/blog/colour-scales.md

```diff
@@ -0,0 +1,72 @@
+
+To solve these issues, we implemented a set of options allowing the user to control the colour range dynamically:
+
+<img src="/img/cscale_options.png" alt="Colour scale user options" style="width:500px;"/>
+
+These are:
+
+ - Static
+    - Default: use range from metadata
+    - Custom: user selects arbitrary values for range
+ - Dynamic
+    - Entire dataset: Use min and max values in actual (unfiltered) dataset
+    - Filtered dataset: Use min and max value in actual dataset, filtered by the current filter set.
+
+Here's an example of how 'Entire dataset' differs from 'Filtered dataset'. In these two images, the colour scale fits the entire dataset, hence is consistent across all filter sets. This means there is typically less colour variation for a given set of filters, but since the colours are fixed to the same values across all sets of filters, it is easy to visually compare the values across both filter sets. 
+<img src="/img/cscale_entire_both.png" alt="Entire dataset colour scale"/>
+<img src="/img/cscale_entire_female.png" alt="Entire dataet colour scale"/>
+
+In the next images, the same two filter sets are shown with 'Filtered dataset' selected. The colour scale is fully used for each filter set so the contrast across shades on the map is maximised and colours are easier to distinguish. However, because the 'meaning' of each colour changes across filter sets, different filter sets cannot easily be visually compared. So this may be a more useful option when the user is interested in examining a single filter set.
+<img src="/img/cscale_filtered_both.png" alt="Filtered dataset colour scale"/>
+<img src="/img/cscale_filtered_female.png" alt="Filtered dataet colour scale"/>
+
+With the option to set the range to arbitrary custom values, users can change the colour scale on the fly to suit their purposes. For example, they may wish to manually narrow the range to maximise variation in a particular area of the map:
+<img src="/img/cscale_zoomed_not_user_custom.png" alt=Custom colour scale"/>
+<img src="/img/cscale_zoomed_user_custom.png" alt="Custom colour scale"/>
+
+While researcher-defined default colour ranges remain a good starting point, these additional options improve the utility of Naomi's maps by giving the user more flexibility to explore different aspects of the data, and can reveal insights about particular datasets which could not be predicted by a colour scale defined by generic metadata. 
```

Comment:
Perhaps mention the additional constraint of not wanting to create an [over-complicated interface](https://i.stack.imgur.com/cVWub.gif)

Reply:
Wow, that's a beaut!